src/components/Search/Search.js

```diff
@@ -221,7 +222,7 @@ Search.propTypes = {
    * Handler that runs when an item from the search filter dropdown is selected
    * Returns the selected filter value.
    */
-  onFilterSelect: PropTypes.func.isRequired,
+  onFilterSelect: requiredIf(PropTypes.func, props => props.filters),
```

Comment:
I'm not sure if props here has been merged w/ defaultProps? (If so then this should be filters.length > 0 / or we change defaultProps.filters to undefined.)

Reply:
Great point, `filters.length > 0` should suffice as you can't select a filter if there aren't any